web/src/components/core/InstallationFinished.jsx

```diff
@@ -26,32 +26,61 @@ import {
   EmptyStateBody,
   EmptyStateHeader,
   EmptyStateIcon,
+  ExpandableSection,
+  Hint,
+  HintBody,
 } from "@patternfly/react-core";
 
-import { Page } from "~/components/core";
+import { If, Page } from "~/components/core";
 import { Center, Icon } from "~/components/layout";
 import { useInstallerClient } from "~/context/installer";
+import { EncryptionMethods } from "~/client/storage";
 import { _ } from "~/i18n";
 
+const TpmHint = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Hint className="tpm-hint">
+      <HintBody>
+        <ExpandableSection
+          isExpanded={isExpanded}
+          onToggle={() => setIsExpanded(!isExpanded)}
+          toggleText={
+            <strong>
+              {_("Make sure to boot directly to the new system to finish the configuration of TPM-based decryption.")}
+            </strong>
+          }
+        >
+          <Text>
+            {_("The final step to configure the TPM to automatically open devices will take place during the first boot of the new system. For that to work, the machine needs to boot directly to the new boot loader. If a local media was used to run this installer, make sure is removed before next boot.")}
```

Comment:
What a long line :stuck_out_tongue_winking_eye:

Reply:
Do we have any rule about avoiding them?

For example, in the Ruby code we always break long translatable texts using `\`. Do we have any similar guideline for long texts in `jsx` that we want to keep as a single string for translations?

Reply:
We use the `\` too, although it is ugly because we have to "break" the indentation. See an example at  https://github.com/openSUSE/agama/blob/df72d49fed7e05463d2e0666d4676144c45d8512/web/src/components/storage/VolumeForm.jsx#L410C14-L411

That said, I have no strong preference. Sorry for the NP comment.

Reply:
NP: I am not sure if the tip is clear enough. I mean "Make sure to boot directly to the new system ...". Me as user can ask "why am I not going to boot directly? I only have to reboot".

I would try to give a more explicit message, For example:

~~~
To finish the configuration of the TPM-based decryption, the new installed system must be directly run in the next boot:

* If an installation media was used for running this installer, then remove it before booting.
* If in your boot menu contains several options, then make sure you select the correct system.
~~~

And the details could explain what happens if the configuration is not correctly finished.

Reply:
Good point, but... 


> And the details could explain what happens if the configuration is not correctly finished.

Reinforce my thoughts that tat "expandable hint" component does not fit good here. 

If your suggestion (which I like) is meant to be also the "link" to see/hide these details... that's definetly too much.

Reply:
Note that we already knew that we were somehow miss-using that component there but having in mind that next iterations should take care of that page. But if we're going to extend that message so long... it deserves a better design now.